specutils/io/default_loaders/hst_cos.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+from astropy.io import fits
+from astropy.units import Unit
+from astropy.nddata import StdDevUncertainty
+
+from specutils.io.registers import data_loader
+from specutils import Spectrum1D
+
+__all__ = ['cos_identify', 'cos_spectrum_loader']
+
+
+def cos_identify(origin, *args, **kwargs):
+    """Check whether given file contains HST/COS spectral data."""
+    with fits.open(args[0]) as hdu:
+        if hdu[0].header['TELESCOP'] == 'HST' and hdu[0].header['INSTRUME'] == 'COS':
+           return True
+
+    return False
+
+
+@data_loader(label="HST/COS", identifier=cos_identify, extensions=['FITS', 'FIT', 'fits', 'fit'])
+def cos_spectrum_loader(file_name, **kwargs):
+    """
+    Load file from COS spectral data into a spectrum object.
```

Comment:
This needs to be more specific.  There are lots of "COS files" - it should say exactly *which* COS file (I suspect it's the final result from the pipeline - e.g. what comes out of MAST, right?  If so, say that.)